src/cogent3/util/misc.py

```diff
@@ -130,10 +131,20 @@ def bytes_to_string(data):
     return data
 
 
+def zip_open(filename, mode="r", **kwargs):
+    """open a zip-compressed file"""
+    if mode == "rt":
+        mode = "r"
+    with ZipFile(filename) as zf:
```

Comment:
I think we should assert, on opening a zip that it contains a single member. If not, raise a ValueError.